### ISSUE TITLE
[Forge 1.21 - 1.2.1] Crashes on dedicated server - java.lang.NoClassDefFoundError: net/minecraft/client/gui/components/Renderable

### ISSUE THREAD

**Issue by TigerWalts** [jahirxtrap/walljump#17]
Crashes when running the `TXFConfig.init` method:

https://mclo.gs/GIo7Jh7

```
Details:
	Caused by 0: java.lang.NoClassDefFoundError: net/minecraft/client/gui/components/Renderable
		at java.base/java.lang.ClassLoader.defineClass1(Native Method) ~[?:?] {}
		at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1027) ~[?:?] {}
		at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150) ~[?:?] {}
		at cpw.mods.securejarhandler/net.minecraftforge.securemodules.SecureModuleClassLoader.readerToClass(SecureModuleClassLoader.java:491) ~[securemodules-2.2.19.jar!/:?] {}
		at cpw.mods.securejarhandler/net.minecraftforge.securemodules.SecureModuleClassLoader.findClass(SecureModuleClassLoader.java:406) ~[securemodules-2.2.19.jar!/:?] {}
		at cpw.mods.securejarhandler/net.minecraftforge.securemodules.SecureModuleClassLoader.loadClass(SecureModuleClassLoader.java:422) ~[securemodules-2.2.19.jar!/:?] {}
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?] {}
		at TRANSFORMER/walljump@1.21-1.2.1-forge/com.jahirtrap.walljump.WallJumpMod.<init>(WallJumpMod.java:25) ~[walljump-1.21-1.2.1-forge.jar!/:1.21-1.2.1-forge] {re:classloading}
		at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62) ~[?:?] {}
		at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502) ~[?:?] {}
		at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486) ~[?:?] {}
		at LAYER PLUGIN/net.minecraftforge.javafmlmod@1.21-51.0.24/net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:66) ~[javafmllanguage-1.21-51.0.24.jar!/:51.0.24] {}
		at LAYER PLUGIN/net.minecraftforge.fmlcore@1.21-51.0.24/net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$5(ModContainer.java:125) ~[fmlcore-1.21-51.0.24.jar!/:1.0] {}
```

The Java 21 ClassLoader is loading some of the classes used by fields in TXFConfig. Some of these are types within `net.minecraft.client`.

Depending on how NeoForge and Fabric versions initialize mods, they may also be affected.

**Comment by jahirxtrap** [comment]
This is an error that only occurs with Forge. If it has no solution, I will end Forge support.

**Comment by jahirxtrap** [comment]
Fixed in 1.2.2
